### ISSUE TITLE
Refactor code for finding the retrievability

### ISSUE THREAD

**Issue by user1823** [open-spaced-repetition/fsrs4anki-helper#197]
Currently, the helper add-on uses codes similar to the following for calculating the retrievability.
https://github.com/open-spaced-repetition/fsrs4anki-helper/blob/eccee05311128f06913cefca12386295bfb6ca5c/reschedule.py#L283

This can be improved as follows:
- Change the above line (and similar lines in all the other files) to:
```py
r = forgetting_curve(elapsed_days, s)
```
- In the utils.py, use
```py
def forgetting_curve(elapsed_days, stability):
    return 0.9 ** (elapsed_days / stability) if version[0] == 3 else (1 + elapsed_days / (9 * stability)) ** -1 
```

**Comment by L-M-Sherlock** [comment]
Where is the version from in the utils.py?

**Comment by user1823** [comment]
> Where is the version from in the utils.py?

I have no idea. I just saw that there was a lot of repetition and felt that it is probably possible to reduce it. If it is not possible to do this, you can close this issue.

**Comment by user1823** [comment]
On a unrelated note, I wanted to take a small help from you. 

I rated a card "Again" by mistake and then synced and performed many other actions. So, the "Undo Answer Card" option is no longer available.

I read [this](https://www.reddit.com/r/Anki/comments/d780ko/comment/f120bki/?context=10000), which describes how to remove a rating from the review history.

I modified it to remove the ratings made after 00:00 today. But, this would remove the ratings from all the cards. How do I make this apply to a particular card? 

```py
mw.col.db.execute("delete from revlog where id > 1692469800000")
mw.col.mod_schema(False)
```
